images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -12,7 +12,7 @@ ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR \
     LANG="C.UTF-8"
 
 ARG IMAGES_DIR="/opt/tmp/images"
-RUN oryx prep --skip-detection --platforms-and-versions nodejs=12 --debug \
+RUN oryx prep --skip-detection --platforms-and-versions nodejs=14 --debug \
```

Comment:
Is there a reason we upgrade nodejs version to 14 on the jamstack image?

Reply:
Yes! This is because we use this Jamstack dockerfile for all of the various ostypes. Since nodejs 12 is deprecated, we don't want to force bullseye to have to support node 12 SDKS. Instead, we can upgrade to 14 here, which is available for all of the OS types

Reply:
Ok, thanks for clarifying!